.github/workflows/pages.yml

```diff
@@ -2,12 +2,12 @@
 name: Deploy Jekyll site to Pages
 
 on:
+  pull_request:
+    branches:
+      - "main"
   push:
     branches:
       - "main"
```

Comment:
Remove this, we don't need to filter on the main branch only.

Reply:
Done